Need to add reference to Swashbuckle.AspNetCore (#32451)
dotnet/AspNetCore.Docs: aspnetcore/fundamentals/minimal-apis/openapi.md

* `Microsoft.AspNetCore.OpenApi` is explained in the next section.
* <xref:Microsoft.Extensions.DependencyInjection.EndpointMetadataApiExplorerServiceCollectionExtensions.AddEndpointsApiExplorer%2A> : Configures the app to use the API Explorer to discover and describe endpoints with default annotations. `WithOpenApi` overrides matching, default annotations generated by the API Explorer with those produced from the `Microsoft.AspNetCore.OpenApi` package.

> * `UseSwagger`adds the [Swagger middleware](xref:tutorials/get-started-with-swashbuckle#add-and-configure-swagger-middleware). You need `Swashbuckle.AspNetCore` nuget package to use this command.

Reviewer: ```suggestion
* `UseSwagger`adds the [Swagger middleware](xref:tutorials/get-started-with-swashbuckle#add-and-configure-swagger-middleware). Requires the `Swashbuckle.AspNetCore` nuget package.
```
